src/protosList.json

```diff
@@ -1,5 +1,7 @@
 [
   "google/api/annotations.proto",
+  "google/api/apikeys/v2/apikeys.proto",
```

Comment:
Do the two `apikeys/v2` protos belong here? For me it's some random API, not a "common" proto.

Reply:
I reverted it to main, but this actually happened as a side effect of running listProtos when testing. I'm not sure exactly what that means...